### ISSUE TITLE
[Suggestion] Config that allows non overworld dimensions to be not affected by the entire frozen thing

### ISSUE THREAD

**Issue by DegeneracyIsNotOK** [alcatrazEscapee/primal-winter#15]
By the "entire frozen thing" I mean the: snowfall effect, snow layer pile up, fog and ice spike/ice terrain generation and all other stuff that achieves the frozen look of the mod.

In the nether if you don't plan on going above the bedrock layer, having snow is pointless. Allowing people to disable it in the nether would probably squeeze in a few extra frames as there would be no snow piling up above you constantly and ice spikes generating.

In the end it is up to personal preference, what I have **personally** noticed is that white snow particles falling in a straight line really do contrast quite hard against a dark purple background, and that is why I would **personally** like to have the ability to disable it.

In summary more customizations choices on how you want your world to be affected by the mod.

In a addition, I would suggest a snowy variant of endstone, since the end looks quite messy as it is currently.

_(There is a chance that I have missed the obvious config for this even though i've looked for it a few times already, if this is the case please do make the config more obvious.)_

**Comment by alcatrazEscapee** [comment]
In all cases, the intended behavior is that all winter-y modifications are restricted on a per-biome basis, which by default excludes both nether and end biomes.

Please provide the:
- Minecraft version
- Mod version
- Loader (Fabric / Forge)

so I can look into what is misbehaving.

**Comment by NotoriusQ** [comment]
Could you make a toggle whitelist/blacklist for the config?
If we wanted to add snow to the nether, for some silly reason, ya know.
Or if we just wanted to make it so tundra and mountains had snowstorms, it would be easier than blacklisting all the other biomes.
